packager/Dockerfile.api

```diff
@@ -0,0 +1,10 @@
+# syntax = devthefuture/dockerfile-x
```

Comment:
Is there any way to get around using custom syntax?

Reply:
There is. However, getting around the build step is indeed a lot of work that can be avoided. That's why I ended up going for the dockerfile-x package.